readthedocs/doc_builder/python_environments.py

```diff
@@ -417,9 +434,13 @@ def setup_base(self):
                     'project': self.project.slug,
                     'version': self.version.slug,
                     'msg': 'Removing existing conda directory',
-                }
+                },
             )
             shutil.rmtree(version_path)
+
+        if self.project.has_feature(Feature.UPDATE_CONDA_STARTUP):
```

Comment:
This should also check if the project is using conda, so we don't run it if the user isn't using conda.

Reply:
The code is under the `Conda` class, not need to add a check here.

Reply:
Santos is right.